src/components/common.js

```diff
@@ -321,6 +321,15 @@ export const methodLink = (config, firecloudRoot, dockstoreRoot) => {
   return sourceRepo === 'agora' ? `${firecloudRoot}/#methods/${methodNamespace}/${methodName}/${methodVersion}` : `${dockstoreRoot}/workflows/${methodPath}`
 }
 
+/**
+ * WARNING: Be very careful when using custom renderers because they may override marked's built-in
+ * content sanitization.
+ * @param children markdown content; must be a string
```

Comment:
See [the jsdoc docs](http://usejsdoc.org/tags-param.html) for how to specify the type of a param.